specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1124 @@
+            0.2574014365673065,
+            0.269286036491394,
+            0.2637230455875397,
+            0.2590833008289337,
+            0.2597166895866394,
+            0.2597350180149078
+          ],
+          "items": {
+            "type": "number"
+          }
+        },
+        "latenciesInSeconds": {
+          "type": "array",
+          "example": [
+            0.3398594856262207,
+            0.3659665584564209,
+            0.3736064434051513,
+            0.3513407707214355,
+            0.3370304107666015,
+            0.3187627792358398,
+            0.3283309936523437,
+            0.3503587245941162,
+            0.3080024719238281,
+            0.3327946662902832
+          ],
+          "items": {
+            "type": "number"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "VariableState": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "Variable name."
+        },
+        "filledNARatio": {
+          "type": "number",
+          "description": "Proportion of NaN values filled of the variable.",
+          "minimum": 0.0,
+          "maximum": 1.0
+        },
+        "effectiveCount": {
+          "type": "integer",
+          "description": "Number of effective points counted."
+        },
+        "firstTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "First valid timestamp of the variable."
+        },
+        "lastTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last valid timestamp of the variable."
+        }
+      },
+      "type": "object"
+    },
+    "Model": {
+      "required": [
+        "createdTime",
+        "lastUpdatedTime",
+        "modelId"
+      ],
+      "properties": {
+        "modelId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Model identifier.",
+          "example": "45aad126-aafd-11ea-b8fb-d89ef3400c5f"
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time (UTC) when the model was created.",
+          "example": "2020-06-30T00:00:00Z"
+        },
+        "lastUpdatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time (UTC) when the model was last updated.",
+          "example": "2020-06-30T00:00:00Z"
+        },
+        "modelInfo": {
+          "$ref": "#/definitions/ModelInfo"
+        }
+      },
+      "type": "object",
+      "description": "Response of getting a model."
+    },
+    "ModelList": {
+      "required": [
+        "currentCount",
+        "maxCount",
+        "models"
+      ],
+      "properties": {
+        "models": {
+          "type": "array",
+          "description": "List of models.",
+          "items": {
+            "$ref": "#/definitions/ModelSnapshot"
```

Comment:
This should be a list of "Model" and not "ModelSnapshot". Why is a different schema used here? This is not RESTful.

Reply:
You can use Model and just omit whatever optional properties you need to to keep the size small.

Reply:
Fixed.